### PR TITLE
fix(react-ui-kit): Fix release packaging

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: packages/react-ui-kit/package.json [hunk 1]
```diff
@@ -26,11 +26,24 @@
...
+    "Modal.js",
+    "Progress.js",
+    "Text.js"
+  ],
   "name": "@wireapp/react-ui-kit",
   "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/react-ui-kit",
   "scripts": {
-    "build": "babel src --out-dir dist --no-comments && node scripts/prepare.js",
+    "build":
+      "babel alias --out-dir ./ --no-comments && babel src --out-dir dist --no-comments && node scripts/prepare.js",
```
Comment (AndyLnd) [inline packages/react-ui-kit/package.json]: shouldn't the `&& node scripts/prepare.js` be removed as well?

**Comment by bennyn** [comment]
Am I right that with this PR you would need to do the following when you create a new React UI Kit component?

1. code it
1. Exclude it from `.gitignore`
1. Create an `alias`
1. List it in `files` of `package.json`

**Comment by Yserz** [comment]
It doesn't apply to every single component but every component group.

**Comment by bennyn** [comment]
Instead of doing that can we maybe switch the working directory on `prepublish`?
 https://stackoverflow.com/questions/30286498/change-working-directory-for-npm-scripts

**Comment by Yserz** [comment]
While this works when running directly with NPM this is a problem when executing publish with lerna. In addition to that we would have to overwrite the NPM publish command which doesn't result in the expected behaviour of actually overwriting it. Instead it gets executed after the execution of the internal NPM publish which in turn results in a double publish.

**Pull request merged**